### ISSUE TITLE
[BUG] Weird Space or unknown letter when resending a message with arrow up

### ISSUE THREAD

**Issue by PabloRNC** [SevenTV/Extension#1023]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### This issue exists in the latest nightly version

- [X] I am using the latest nightly version

### What browsers are you seeing the problem on?

Opera

### Current Behavior

Hi. I was developing over the TwitchAPI and I found an strange bug with the Opera extension of 7TV.

When I was sending a message like `hello` and then with the `up arrow` I recovered the message (so for not writting hello everytime) that message was sent with a weird character or space at the end. 

I say a strange character because after somes test that character isn't a space at all.

Here is an example about it:

When I send hello for the first time:

`hello`

And then with that weird character:

`hello 󠀀`

![image](https://github.com/SevenTV/Extension/assets/99443080/0bf4e72b-5588-43c9-85d5-28be5960cb9a)
![image](https://github.com/SevenTV/Extension/assets/99443080/912e002e-93c0-4dc0-bd46-90385b76ffa0)

As you can see there is a slight difference in a character at the end of the string which currently as I tested it's not a whitespace at all since sometimes I can remove it an others no.

### Expected Behavior

The message which I re-sent should be the same as the first one when recovering it with the up arrow and then press enter to re send it without writing it

### Steps To Reproduce

Send any message to a chatroom.

Press arrow up and recover that message.

Re-send it.

**Comment by PabloRNC** [comment]
I want to add this image.

As you can see there are some weird characters as I said. The first char is what it would be if there are only white spaces but in the second, the char that was sent by 7TV on the second message, there are weird characters.

![image](https://github.com/SevenTV/Extension/assets/99443080/0bc38cdc-9744-4e3f-aef3-ee998cdf502c)

**Comment by PabloRNC** [comment]
![image](https://github.com/SevenTV/Extension/assets/99443080/5478e455-2b12-4a32-917f-133c9b331a7a)

**Comment by PabloRNC** [comment]
In the clipboard also

**Comment by pimothyxd** [comment]
Do you have the `Bypass Duplicate Message Check` setting on by chance? If so, it's behavior is intentional. It works by adding said space to the message to avoid duplicates. https://github.com/SevenTV/Extension/blob/a98e3653a0d0714f945e57d20daf53f2cf5834f0/src/site/twitch.tv/modules/chat-input/ChatSpam.vue#L48-L66

**Comment by PabloRNC** [comment]
Okey but as I send is not such a whitespace

**Comment by PabloRNC** [comment]
It's a weird character

**Comment by pimothyxd** [comment]
That is correct as the message is modified by the extension before it is sent.

https://github.com/SevenTV/Extension/blob/a98e3653a0d0714f945e57d20daf53f2cf5834f0/src/common/Constant.ts#L20

**Comment by PabloRNC** [comment]
So I could split or parse that character with that. 


Thanks. 

I couldn't find what it was.
